recipe/meta.yaml

```diff
@@ -47,4 +47,6 @@ about:
 
 extra:
   recipe-maintainers:
-    - jp
+    - kujaku11
+    - kkappler
+    - lheagy
```

Comment:
This reverts an accidental change [here]( https://github.com/conda-forge/mt-metadata-feedstock/commit/08100fcf411e0c938d6f81b30ab3288338d2d6c9#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR48-R50 )